pkg/webhooks/policymutation.go

```diff
@@ -66,20 +66,18 @@ func generateJSONPatchesForDefaults(policy *kyverno.ClusterPolicy, operation v1b
 		updateMsgs = append(updateMsgs, updateMsg)
 	}
 
-	// TODO(shuting): enable this feature on policy UPDATE
-	if operation == v1beta1.Create {
```

Comment:
Need to cover `kubectl edit`.

Reply:
@realshuting requested changes have been made